wallet-connect-test-bench/front-end/src/writing_to_blockchain.ts

```diff
@@ -24,8 +33,61 @@ import {
     SET_TIMESTAMP_PARAMETER_SCHEMA,
     SET_STRING_PARAMETER_SCHEMA,
     SET_OPTION_PARAMETER_SCHEMA,
+    BASE_64_TEST_BENCH_SMART_CONTRACT_MODULE,
+    BASE_64_SCHEMA_WITH_INIT_U8_PARAMETER,
 } from './constants';
 
+export async function initializeWithoutAmountWithoutParameter(connection: WalletConnection, account: string) {
+    return connection.signAndSendTransaction(account, AccountTransactionType.InitContract, {
+        amount: new CcdAmount(BigInt(0)),
+        moduleRef: new ModuleReference('ca34210298992899b85a45d8c639a4442df25ebb3c4360da671d766f8fb079e2'),
+        initName: 'smart_contract_test_bench',
+        param: toBuffer(''),
+        maxContractExecutionEnergy: 300000n,
+    } as InitContractPayload);
+}
+
+export async function initializeWithAmount(connection: WalletConnection, account: string) {
+    return connection.signAndSendTransaction(account, AccountTransactionType.InitContract, {
+        amount: new CcdAmount(BigInt(1000000)),
+        moduleRef: new ModuleReference('da913dbdaebac0fc3be60d61adcabb000a22e580a4aecdf70985cff6f4f55360'),
+        initName: 'smart_contract_test_bench',
+        param: toBuffer(''),
+        maxContractExecutionEnergy: 300000n,
```

Comment:
That's a bit much and will mean you have to have quite a lot of CCD on the account to test it.